skycoin/dmsg
# Improve shutdown server logic

Fixes #170 

 Changes:	
- Remove update entry on delete session from server side

How to test this PR:
- Run a dmsg-server based on this PR branch on online instance with `availableSession=3000` in config.
- Clone skywire, checkout to develop, add `time.Sleep(15*time.Minute)` at the beginning of `initLauncher` method at `pkg>visor>init.go` file. Build binaries.
- Copy `underload.sh` from integration folder to skywire cloned folder.
- Remove `sed ...` command line (line 23) in `underload.sh.
- Run tons of visor by `bash underload.sh -n 200 -u dmsgd.skywire.dev` there.
- After all visor connect to dmsg-server, shutdown it with `Ctrl+C`. 
- Should shutdown immediately. 

Note:
I remove update entry of dmsg server when deleting a session. In server shutdown no problem occurs because no need update data in fact. And in disconnect a client (visor) from server, this dmsg server entry update done in internal update at max in next 10 seconds. So no problem occurs here too. Update entry remains same as before for new sessions.
